integration/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "integration-tests",
+  "version": "1.0.0",
+  "description": "Integration testing for Amazon Chime SDK React Components Library",
+  "scripts": {
+    "test": "./script/run-test",
+    "test:fast": "mocha ./test/*"
```

Comment:
what is this option for? Doesn't seem it is faster than the default option?

Reply:
The `test` script starts the demo, run the test, check the result and terminate the demo.
The `test:fast` just runs the test directly, it's mostly for development and debugging.
So when you develop/debug the test, you just need to start the test demo your self and run this script.

Reply:
but does just run all tests?

Reply:
It only runs all integration tests. Since this `package.json` file is in the integration folder, I think there's no need to name them as `test:integration` or `test:integration-fast`. There should be enough context here, I assume.